app/models.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Actor:
+    first_name: str
+    last_name: str
+    id: int
```

Comment:
Ensure that the `id` attribute is correctly handled when creating `Actor` instances from database rows. The order of attributes in the dataclass should match the order of columns in the database queries.